### DISCUSSION TITLE
Cannot map a directory using lima & nerdctl in macOS

### DISCUSSION THREAD

**Discussion by j0nl1** [containerd/nerdctl#862] [Q&A]
I tried to create a mapping from mongodb but the files aren't copy to my local folder.
I've tried using pwd and ~/ (I'm using docker as alias)

is there something else I should config?

![image](https://user-images.githubusercontent.com/40375118/156861111-73246688-33d5-4137-bac0-a4e70dfacec7.png)

**Comment by AkihiroSuda** [comment]
Lima mounts the host home with read-only by default.
You can use `limactl edit default` to change it to be writable 
https://github.com/lima-vm/lima/blob/66f66e1100e9fffe0969aa8d1f42ec7b39d06442/pkg/limayaml/default.yaml#L49

**Reply by j0nl1** [reply]
I already did that before to write the issue. I stopped lima default instance, modified the file config in .lima directory and start

![image](https://user-images.githubusercontent.com/40375118/156865809-be98d8b1-6f57-415a-b802-11e1b5780ed0.png)
